site/layouts/course/pdf.html

```diff
@@ -1,3 +1,10 @@
+{{ define "header" }}
+{{ partial "header" . }}
+{{ partial "header_placeholder.html" . }}
+{{ end }}
+{{ define "subheader" }}
+{{ partial "chp_partial.html" (dict "partial" "course_banner.html" "context" .) }}
+{{ end }}
```

Comment:
can we just change this in the course `baseof.html` template instead? I just added this line to `course/baseof.html` and the header is on all course pages:

```
    {{ block "header" . }}{{ partial "header" . }}{{ partial "header_placeholder.html" . }}{{ end }}
```